infrastructure/builder/src/generate/generators/docs-tocs.js

```diff
@@ -0,0 +1,204 @@
+const fs = require('fs');
+const md = require('md-directory');
+const readDirectory = require('read-directory');
+const { join } = require('path');
+const { readJSONSync, writeJSON } = require('../util');
+const removeExtension = require('../../../../../ui/src/utils/removeExtension');
+
+const DOCS_LOCATIONS = {
+  GENERATED: join('generated', 'docs'),
+  STATIC: join('ui', 'src', 'docs'),
+};
+
+const projectMetadata = {};
```

Comment:
we can avoid a global variable if we define it right in `readGeneratedDocs` and get rid of `readProjectMetadata` function